admin-ui/app/dialogs/send-push-notification.html

```diff
@@ -14,7 +14,16 @@ <h4 class="modal-title" id="myModalLabel">Send push to Application One</h4>
       </div>
     </div>
 
-
+      
```

Comment:
Do you really think the prio needs to be on the dialog? I am not sure if that's really _that_ useful, given that it's probably pretty hard to tell a difference between HIGH/NORMAL when doing a test push :smile: